### ISSUE TITLE
Build error with cnbuilder library

### ISSUE THREAD

**Issue by sopretty** [xobotyi/react-scrollbars-custom#141]
## Type of Issue

```
[x] Bug Report
```
## Description

There is a new release on "cnbuilder" library causing some troubles: 
https://github.com/xobotyi/cnbuilder/blob/master/CHANGELOG.md (see the breaking changes)
They forgot to say that they have deployed a 1.3.0 version with those breaking changes..

```
BUILD Warning in rsc.esm.js 746:22-25
"export 'default' (imported as 'cnb') was not found in 'cnbuilder'
```
And everything breaks when using the scrollbar.

#### How To Reproduce

Create a react project (with create-react-app), install react-scrollbars-custom and build your application

#### Version Information

Deps versions

```
  "react-scrollbars-custom": "~4.0.21"
```
#### Temporary Fix

EDIT: I've fixed it by importing ```{cnb}``` instead of the default export in the Scrollbar.tsx file ```import cnb from "cnbuilder";```.

**Comment by gsi-alejandro-marino** [comment]
I got the same error

**Comment by xobotyi** [comment]
@sopretty retry now please.
had an issues with semantic release.

**Comment by sopretty** [comment]
@xobotyi Thanks for responding that fast !

**Comment by xobotyi** [comment]
@sopretty dang it! sry, just a minute

**Comment by xobotyi** [comment]
@sopretty tested locally with 1.3.2 seems it got fine now.

**Comment by xobotyi** [comment]
fixed with cnbuilder v1.3.2 release

**Comment by sopretty** [comment]
@xobotyi It works well  !
Thank you.
